src/app/pages/storage/volumes/volumes-list/volumes-list.component.ts

```diff
@@ -523,23 +523,20 @@ export class VolumesListTableConfig implements InputTableConf {
           label: T("Delete Dataset"),
           onClick: (row1) => {
 
-            this.dialogService.confirm(T("Delete"), T("This action is irreversible and will \
-             delete any existing snapshots of this dataset (" + row1.path + ").  Please confirm."), false, T('Delete Dataset')).subscribe((res) => {
-                if (res) {
+            this.dialogService.confirm(T("Delete"), 
+              T("This action is irreversible and will delete any existing snapshots of this dataset ") + "(" + row1.path + ")."+  T(" Please confirm.")
```

Comment:
The confirm should already be part of the dialog box, right?  Otherwise, asking the user to "please confirm" implies that they should do it, which is not the safe recommendation.
```suggestion
              T("Delete the dataset " + "<i>" + row1.path + "</i>"+  T(" and all snapshots of it?")
```